web/src/main/webapp/js/main-template.js

```diff
@@ -547,7 +584,11 @@ function routeLatLng(request, doQuery) {
             var geojsonFeature = {
                 "type": "Feature",
                 "geometry": path.points,
-                "properties": {"style": style}
+                "properties": {
+                    "style": style,
+                    name: "route",
+                    snappedWaypoints: path.snapped_waypoints
```

Comment:
I know it is a bit nitpicking but as we 'emulate' GeoJSON we would need underscore - does this make sense :) ?

I.e.
snapped_waypoints: path.snapped_waypoints